lib/_http_outgoing.js

```diff
@@ -514,6 +517,9 @@ function processHeader(self, state, key, value, validate) {
 function storeHeader(self, state, key, value, validate) {
   if (validate)
     validateHeaderValue(key, value);
+  if (key === 'Content-Length') {
```

Comment:
```suggestion
  if (key.toLowerCase() === 'content-length') {
```

Reply:
https://nodejs.org/es/docs/guides/anatomy-of-an-http-transaction/

It's important to note here that all headers are represented in lower-case only, regardless of how the client actually sent them. This simplifies the task of parsing headers for whatever purpose.